src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/TopXLine.java

```diff
@@ -0,0 +1,32 @@
+package org.betonquest.betonquest.compatibility.holographicdisplays;
+
+import lombok.Getter;
+
+/**
+ * Stores entry data of a {@link TopXObject}.
+ */
+@SuppressWarnings("PMD.CommentRequired")
```

Comment:
New Classes should not have any `@SuppressWarnings`, except false positives

And `CommentRequired` is not allowed at all for new Classes.